shell/browser/api/atom_api_web_contents.cc

```diff
@@ -460,7 +455,7 @@ WebContents::~WebContents() {
     } else {
       // Destroy WebContents asynchronously unless app is shutting down,
       // because destroy() might be called inside WebContents's event handler.
-      DestroyWebContents(true /* async */);
+      DestroyWebContents(!IsGuest() /* async */);
```

Comment:
I'm not sure `!IsGuest()` is what we expect here. @zcbenz opinion?

Reply:
It is correct.